lib/folio_record.rb

```diff
@@ -2,8 +2,10 @@
 
 require 'active_support/core_ext/module/delegation'
 require_relative 'traject/common/constants'
+require 'csv'
 
 class FolioRecord
+  LIMIT = (2**31) - 1 # Folio max results
```

Comment:
this is also a magic number here: https://github.com/sul-dlss/searchworks_traject_indexer/blob/master/lib/traject/readers/folio_reader.rb#L19